db/pg/index.js

```diff
@@ -31,13 +39,10 @@ module.exports = exports = async (options = DB) => {
 				host: options.host,
 				password: options.password,
 				port: options.port,
-				user: options.user_name
+				user: options.user_name,
+				...sslSetUp,
```

Comment:
**Question**: Are we expecting the addition to the target object of the following?

from:

```js
{
    ..., 
    ssl: { 
        rejectUnauthorized : false 
    }  
}
```

to: 

```js
{
    ..., 
    ssl: true,
	extra: {
		ssl: {
			rejectUnauthorized: false,
		},
	}
} 
```

Reply:
According to https://github.com/brianc/node-postgres/tree/master/packages/pg-connection-string#usage, the connection object does not expect a `extra: {...}`. I am struggling to find where `extra` could be used.